SpringParking-parkingBranch/src/main/java/pl/springDemo/demo/userPayment/AmountCounter.java

```diff
@@ -0,0 +1,39 @@
+package pl.springDemo.demo.userPayment;
+
+import pl.springDemo.demo.user.UserType;
+
+public class AmountCounter {
+
+    public double countHoursAmount(UserType userType, long hours) {
+        switch (userType) {
+            case VIP_DRIVER:
+                if (hours <= 1) {
+                    return 0;
+                } else if (hours == 2) {
+                    return 2;
+                } else {
+                    for (int i = 0; i < hours; i++) {
+                        double a = 2 * 1.5;
```

Comment:
Czy wynik będzie dokładny? Jak można to poprawić?

Reply:
Tak jak zostało napisane, zdecydowanie przydałoby się rzeczywiście więcej testów, które mogłyby sprawdzić dokładność tej metody.